experiment/autobumper/main.go

```diff
@@ -174,7 +174,7 @@ func main() {
 		logrus.WithError(err).Fatal("Failed to push changes.")
 	}
 
-	if err := bumper.UpdatePR(gc, githubOrg, githubRepo, images, getAssignment(), "Update prow to", o.githubLogin+":"+remoteBranch, "master"); err != nil {
+	if err := bumper.UpdatePR(gc, githubOrg, githubRepo, images, getAssignment(), "Update prow to", o.githubLogin+":"+remoteBranch, "master", true); err != nil {
```

Comment:
Consider using a const here to improve readability
```golang
const disallowMods = false
if err := bumper.UpdatePR(gc, githubOrg, ..., disallowMods) {
```

(note that this is how updater.go worked before)